src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -41,18 +42,42 @@ const ExplorationViewPage = ({
   const { windows, panel } = layout;
   const [selectedTab, setSelectedTab] = useState(panel);
   const [addMenuVisible, setAddMenuVisible] = useState(false);
-
+  const { method } = useSelector((state) => (
+    state.experimentSettings.processing?.configureEmbedding?.embeddingSettings || ' '
+  ));
   useEffect(() => {
     setSelectedTab(panel);
   }, [panel]);
 
+  useEffect(() => {
+    if (!method) {
+      dispatch(loadProcessingSettings(experimentId));
+    }
+  }, []);
+  const methodUppercase = method ? method.toUpperCase() : ' ';
+
+  useEffect(() => {
+    if (method && windows) {
+      dispatch(updateLayout({
+        ...windows,
+        first: {
+          ...windows.first,
+          first: {
+            ...windows.first.first,
+            first: `${methodUppercase} Embedding`,
+          },
+        },
+      }));
+    }
+  }, [method]);
+
+  const embeddingTileName = `${methodUppercase} Embedding`;
```

Comment:
I suggest you rename this `embeddingTileName` to `embeddingTitle` and move it to be defined on line 58, then use it also in line 67.